src/main/kotlin/com/chattriggers/ctjs/api/message/TextComponent.kt

```diff
@@ -217,6 +218,26 @@ class TextComponent : Text {
         setHover(HoverEvent.Action.byName(action) as HoverEvent.Action<*>, value)
     }
 
+    /**
+     * Sets the color of this [TextComponent]
+     * This won't override your color codes unless &r is explicitly used.
+     *
+     * @param color RGB value acquired using [Renderer.getColor]. Alpha values will be ignored
+     */
+    fun setColor(color: Int) = apply {
```

Comment:
I think using Long would be better here according to similar methods inside Renderer.

Reply:
Even if colored styles only accept RGB colors?